src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparator.java

```diff
@@ -20,9 +20,6 @@
 import java.util.Collections;
 /* no violation: block comment after token*/
 
-import com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator.InputEmptyLineSeparator;
```

Comment:
We need to replace this with another import and trailing single line comment

Reply:
done